editor/editor_node.cpp

```diff
@@ -6248,7 +6248,9 @@ EditorNode::EditorNode() {
 
 	p = settings_menu->get_popup();
 	p->set_hide_on_window_lose_focus(true);
-	p->add_shortcut(ED_SHORTCUT("editor/editor_settings", TTR("Editor Settings...")), SETTINGS_PREFERENCES);
```

Comment:
This line should be kept in an `#else` statement. Otherwise, the **Editor Settings...** option will disappear entirely on non-macOS platforms :no_mouth:

Reply:
Oops, that's my bad, thanks for noticing my noobice, fixing ...